rpc/src/rpc_pubsub_service.rs

```diff
@@ -401,6 +401,12 @@ async fn handle_connection(
             pin!(receive_future);
             loop {
                 select! {
+                    biased; // See [prioritization] note below.
+
+                    // [prioritization]
+                    // This block must come FIRST in the `select!` macro. This prioritizes
+                    // processing received messages over sending messages. This ensures the timely
```

Comment:
I'm assuming notifications to subscribers will now be less timely?

It's not obvious to me why this tradeoff is better. Care to enlighten me?

Reply:
Chatted even more with @steveluscher about this in DMs. He has convinced me this is strictly better.

One important thing I didn't catch at first is the loops is per-websocket